addons/l10n_in_edi/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="res_config_settings_view_form_inherit_l10n_in_edi" model="ir.ui.view">
+        <field name="name">res.config.settings.form.inherit.l10n_in_edi</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="iap.res_config_settings_view_form"/>
+        <field name="arch" type="xml">
+            <div id="iap_portal" position="inside">
```

Comment:
This should be in invoicing, not iap portal

Reply:
okay